sixteenSegmentDisplay.pde

```diff
@@ -1,6 +1,6 @@
-// This programme based on anusha-exe's seven segment display LED clock programme
+// This programme is based on The Coding Train video on Seven Segment Display

```

Comment:
I really don't agree with this particular line, good lady. So if you fix it, I can pull it.